wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -17,40 +17,125 @@ function chld_thm_cfg_locale_css( $uri ){
 if ( !function_exists( 'chld_thm_cfg_parent_css' ) ):
     function chld_thm_cfg_parent_css() {
         wp_enqueue_style( 'chld_thm_cfg_parent', trailingslashit( get_template_directory_uri() ) . 'style.css', array(  ) );
+        wp_enqueue_style( 'chld_thm_cfg_child', trailingslashit( get_stylesheet_directory_uri() ) . 'style.css', array(  ) );
     }
 endif;
 add_action( 'wp_enqueue_scripts', 'chld_thm_cfg_parent_css', 10 );
 
-function add_my_filter($content) {
-    $content = "<div>This is my filter</div>".$content;
+function add_my_filter( $content ) {
+    $content = _e( apply_filters( 'custom_filter_change', "This is my filter" )) . $content;
     return $content;
 }
-add_filter( 'the_content', 'add_my_filter', 10 );
+//add_filter( 'the_content', 'add_my_filter', 10 );
 
-function add_two($content) {
+function change_my_filter() {
+    return "<div>This is my extendable filter</div>";
+}
+//add_filter( 'custom_filter_change', 'change_my_filter', 10 );
+
+function add_two( $content ) {
     $content .= "<div>Two</div>";
     return $content;
 }
-add_filter( 'the_content', 'add_two', 11 );
+//add_filter( 'the_content', 'add_two', 11 );
 
-function add_one($content) {
+function add_one( $content ) {
     $content .= "<div>One</div>";
     return $content;
 }
-add_filter( 'the_content', 'add_one', 10 );
+//add_filter( 'the_content', 'add_one', 10 );
 
-function add_three($content) {
+function add_three( $content ) {
     $content .= "<div>Three</div>";
     return $content;
 }
-add_filter( 'the_content', 'add_three', 11 );
+//add_filter( 'the_content', 'add_three', 11 );
 
-function add_custom_menu_item ($items, $args) {
+function add_custom_menu_item ( $items, $args ) {
     if(is_user_logged_in()) {
-        $items .= '<li><a href="/devrix-starter/wp-admin/profile.php">Profile Settings</a></li>';
+        $items .= apply_filters('menu_items', '');
     }
     return $items;
 }
 add_filter( 'wp_nav_menu_items', 'add_custom_menu_item', 10, 2 );
 
+function add_students_archive( $items ) {
+    $items .= '<li><a href=' . home_url( '/student' ) . '>Students Archive</a></li>';
+    return $items;
+}
+add_filter( 'menu_items', 'add_students_archive' );
+
+function add_profile_settings( $items ) {
+    $items .= '<li><a href=' . admin_url('profile.php') . '>Profile Settings</a></li>';
+    return $items;
+}
+add_filter( 'menu_items', 'add_profile_settings' );
+
+
+function email_on_update( $user_id, $old_user_data ) {
+    $to = get_bloginfo( 'admin_email' );
+    $subject = 'Update';
+    $body = 'There was an update';
+    $headers = array( 'Content-Type: text/html; charset=UTF-8' );
+
+    wp_mail( $to, $subject, $body, $headers );
+}
+add_action( 'profile_update', 'email_on_update', 10, 2 );
+
+function student_custom_box_html( $post ) {
+    ?>
+    <form method="post">
```

Comment:
we might not need a `<form>` here + inline css

+ value fields are empty, I guess that's gonna be updated with the next PR?